src/waveresponse/_standardized1d.py

```diff
@@ -0,0 +1,397 @@
+        in terms of Hz and rad/s, respectively.
+        """
+        return super().__call__(A, B, freq_hz=freq_hz)
+
+    def _spectrum(self, omega, A, B):
+        return A / omega**5.0 * np.exp(-B / omega**4)
+
+
+class ModifiedPiersonMoskowitz(BasePMSpectrum):
+    """
+    Modified Pierson-Moskowitz (i.e., Bretschneider) spectrum, given by:
+
+    ``S(w) = A / w ** 5 exp(-B / w ** 4)``
+
+    where,
+
+    - ``A = 5 / 16 * Hs ** 2 * w_p ** 4``
+    - ``B = 5 / 4 * w_p ** 4``.
+    - ``Hs`` is the significant wave height.
+    - ``w_p = 2pi / Tp`` is the angular spectral peak frequency.
+
+    Parameters
+    ----------
+    freq : array-like
+        Sequence of frequencies to use when generating the spectrum.
+    freq_hz : bool
+        Whether the provided frequencies are in rad/s (default) or Hz.
+
+    See Also
+    --------
+    JONSWAP : JONSWAP wave spectrum.
+    OchiHubble : Ochi-Hubble (three-parameter) wave spectrum.
+    """
+
+    def __call__(self, hs, tp, freq_hz=None):
+        """
+        Generate wave spectrum.
+
+        Parameters
+        ----------
+        hs : float
+            Significant wave height, Hs.
+        tp : float
+            Peak period, Tp.
+        freq_hz : bool, optional
+            Whether to return the frequencies and spectrum in terms of rad/s (`True`)
+            or Hz (`False`). If `None` (default), the original units of `freq` is
+            preserved.
+
+        Return
+        ------
+        freq : 1-D array
+            Frequencies corresponding to the spectrum values. Unit is set according
+            to `freq_hz`.
+        spectrum : 1-D array
+            Spectrum values. Unit is set according to `freq_hz`.
+
+        Notes
+        -----
+        The scaling between wave spectrum in terms of Hz and rad/s is defined
+        as:
+
+        ``S(f) = 2*pi*S(w)``
+
+        where ``S(f)`` and ``S(w)`` are the same spectrum but expressed
+        in terms of Hz and rad/s, respectively.
+        """
+
+        A = self._A(hs, tp)
+        B = self._B(hs, tp)
+
+        return super().__call__(A, B, freq_hz=freq_hz)
+
+    def _A(self, *args):
+        hs, tp = args
+        omega_p = 2.0 * np.pi / tp
+        return (5.0 / 16.0) * hs**2.0 * omega_p**4.0
+
+    def _B(self, *args):
+        _, tp = args
+        omega_p = 2.0 * np.pi / tp
+        return (5.0 / 4.0) * omega_p**4
+
+
+class JONSWAP(ModifiedPiersonMoskowitz):
+    """
+    JONSWAP spectrum, given as:
+
+    ``S(w) = alpha * S_pm(w) * gamma ** b``
+
+    where,
+
+    ``b = exp(-(w - w_p) ** 2 / (2 * sigma ** 2 * wp ** 2))``
+
+    and,
+
+    - ``S_pm(w)`` is the Pierson-Moskowitz (PM) spectrum.
+    - ``gamma`` is a peak enhancement factor.
+    - ``alpha = 1 - 0.287 * ln(gamma)`` is a normalizing factor.
+    - ``sigma`` is the spectral width parameter:
+        - ``sigma = simga_a`` for ``w <= wp``
+        - ``sigma = sigma_b`` for ``w > wp``
+    - ``wp = 2pi / Tp`` is the angular spectral peak frequency.
```

Comment:
`alpha` depends on `sigma`. If `alpha` is hard-coded but `sigma` is user-defined, then `Hs` is not preserved.

I think we should hard-code sigma values as well. If anyone needs to modify the common JONSWAP then they have to construct it themselves.